symmetricTree.cpp

```diff
@@ -0,0 +1,36 @@
+// Time Complexity : O(n)
+// Space Complexity : O(1)
```

Comment:
Recursive stack space should also be considered auxiliary. So, space complexity would be O(H) here